src/combinatorics/colex.lean

```diff
@@ -181,14 +181,26 @@ end
 
 instance [linear_order α] : is_trichotomous (finset.colex α) (<) := ⟨lt_trichotomy⟩
 
--- It should be possible to do this computably but it doesn't seem to make any difference for now.
-noncomputable instance [linear_order α] : linear_order (finset.colex α) :=
+/-- Show that `colex.lt` is decidable. This isn't an instance since it's re-inferred from the
+(computable) `decidable_le` instance below. -/
```

Comment:
I'd recommend just making this an instance and adding `decidable_lt := by apply_instance` below.

Reply:
Is it worth it?

Reply:
It might save an @, and removes the need for this comment

Reply:
Changed, thanks.